src/fsharp/service/ServiceAnalysis.fs

```diff
@@ -241,58 +245,58 @@ module SimplifyNames =
 
     let getPlidLength (plid: string list) = (plid |> List.sumBy String.length) + plid.Length    
 
-    let getSimplifiableNames (checkFileResults: FSharpCheckFileResults, getSourceLineStr: int -> string) : Async<SimplifiableRange list> =
+    let getSimplifiableNames (checkFileResults: FSharpCheckFileResults, getSourceLineStr: int -> string) =
         async {
             let result = ResizeArray()
             let! ct = Async.CancellationToken
-            let symbolUses = checkFileResults.GetAllUsesOfAllSymbolsInFile(ct)
+            let filter = fun (symbolUse: FSharpSymbolUse) -> not symbolUse.IsFromOpenStatement && not symbolUse.IsFromDefinition
```

Comment:
Again prefer 

```
                checkFileResults.GetAllUsesOfAllSymbolsInFileByPredicate ct 
                    (fun symbolUse -> 
                         not symbolUse.IsFromOpenStatement && not symbolUse.IsFromDefinition
                     ) 
               |> Array.Parallel.choose (fun symbolUse ->
```

rather than `let filter = fun ...`